policyengine_api/routes/household_routes.py

```diff
@@ -0,0 +1,137 @@
+from flask import Blueprint, Response, request
+from werkzeug.exceptions import NotFound, BadRequest
+import json
+
+from policyengine_api.services.household_service import HouseholdService
+from policyengine_api.utils.payload_validators import (
+    validate_household_payload,
+    validate_country,
+)
+
+
+household_bp = Blueprint("household", __name__)
+household_service = HouseholdService()
+
+
+@household_bp.route(
+    "/<country_id>/household/<int:household_id>", methods=["GET"]
+)
+@validate_country
+def get_household(country_id: str, household_id: int) -> Response:
+    """
+    Get a household's input data with a given ID.
+
+    Args:
+        country_id (str): The country ID.
+        household_id (int): The household ID.
+    """
+    print(f"Got request for household {household_id} in country {country_id}")
```

Comment:
*discussion, nonblocking* - 
Suggest this as an optimization after introducing  "real" logging.

In the theme from yesterday of making our route functions as clean and simple as possible, logging is another function the framework should be able to do for us in many cases.

In this case the blueprint already defines what the path is, the names of the parameters, the method type and has the parsed values from the path, etc. so it should be possible to automate logging all the relevant "got a request" information.

1. Reduces clutter/duplication
2. reduces cases where we forget to manually add the log message
3. makes it easy to add more detail later and get that for every method (I.e. maybe we want to log which user is making the request)